### PR TITLE
Test coverage for cache

### PR DESCRIPTION
**Pull request by gabrysiaolsz**
Improve cache test coverage from 67% to 100%

### CODE REVIEW

File: lib/src/test/java/com/google/cloud/datalineage/producerclient/ApiEnablementCacheOptionsTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  public void getDefaultInstance_setsDefaultOptions() {
+    ApiEnablementCacheOptions options = ApiEnablementCacheOptions.getDefaultInstance();
+    assertThat(options.getDefaultCacheDisabledStatusTime())
+        .isEqualTo(ApiEnablementCacheOptions.DEFAULT_DISABLED_TIME);
+    assertThat(options.getCacheSize()).isEqualTo(ApiEnablementCacheOptions.DEFAULT_SIZE);
+    assertThat(options.getClock()).isEqualTo(ApiEnablementCacheOptions.DEFAULT_CLOCK);
+  }
+
+  @Test
+  public void toBuilder() {
```
Comment (ngorchakova) [inline lib/src/test/java/com/google/cloud/datalineage/producerclient/ApiEnablementCacheOptionsTest.java:75]: please rename this test

File: lib/src/test/java/com/google/cloud/datalineage/producerclient/ApiEnablementCacheFactoryTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** * Test suite for ApiEnablementCacheFactory */
+@RunWith(JUnit4.class)
+public class ApiEnablementCacheFactoryTest {
+
+  @Test
+  public void get_disabledSettings_returnsNoOpCache() {
```
Comment (ngorchakova) [inline lib/src/test/java/com/google/cloud/datalineage/producerclient/ApiEnablementCacheFactoryTest.java:28]: [minor] In the tests below you use naming getDefaultInstance_setsDefaultOptions, and here get_.... 
would be good to be consistent

**Review by ngorchakova — APPROVED** [review]

File: lib/src/test/java/com/google/cloud/datalineage/producerclient/ApiEnablementCacheOptionsTest.java (hunk 1 above)
Comment (gabrysiaolsz) [reply lib/src/test/java/com/google/cloud/datalineage/producerclient/ApiEnablementCacheOptionsTest.java:75]: done

File: lib/src/test/java/com/google/cloud/datalineage/producerclient/ApiEnablementCacheFactoryTest.java (hunk 2 above)
Comment (gabrysiaolsz) [reply lib/src/test/java/com/google/cloud/datalineage/producerclient/ApiEnablementCacheFactoryTest.java:28]: done

**Pull request merged**